about.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="./styles/about.css">
+    <title>VueJs | Amsterdam</title>
+</head>
+<body>
+    <nav class="nav-header">
+        <i class="fa fa-facebook nav-icons" aria-hidden="true"></i>
+        <i class="fa fa-twitter nav-icons" aria-hidden="true"></i>
+        <ul class="nav-header-list">
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+        </ul>
+    </nav>
+    <header class="header">
+        <img src="./images/logs.png" alt="" class="logo">
+        <ul class="nav-header-list">
+            <li><a href="./about.html">About</a></li>
```

Comment:
- [ ] Please kindly place an appropriate `alt` attribute on all `img` tags. do this everywhere else you have `img` tags